### PR TITLE
[ch14865] Condition display of PSEA app content by environment flag

### PR DESCRIPTION
**Pull request by danNordlogic**
[ch14865] Condition display of PSEA app content by environment flag

### CODE REVIEW

**Comment by adi130987** [comment]
@robertavram I think we need to add `psea_enabled` flag to BE and make it active before we merge this.

**Comment by robertavram** [comment]
backend flag is added from the admin. if it doesn't exist then it should work by default...

We only disable with the flag.. so that would be a change.

File: src_ts/components/common/environment-flags.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    .catch((err: any) => {logError('[AppShell]', 'checkEnvFlags', err)});
+}
+
+function handleEnvFlagsReceived(envFlags: any) {
+  if (envFlags && envFlags.active_flags && envFlags.active_flags.includes('psea_disabled')) {
+    const bodyEl = document.querySelector('body');
+    if (bodyEl) {
+      bodyEl.querySelectorAll('*').forEach(el => el.remove());
+      const warnBox = document.createElement('etools-error-warn-box') as EtoolsErrorWarnBox;
+      warnBox.messages = ['You are not authorized to use this application'];
```
Comment (robertavram) [inline src_ts/components/common/environment-flags.ts]: @danNordlogic can we please change this message to: 
"PSEA is currently unavailable in your workspace, please stay tuned...  In the meantime checkout our other great modules"

Comment (robertavram) [reply src_ts/components/common/environment-flags.ts]: this comes from @ntrncic

**Pull request merged**